MarsRover/Controller.cs

```diff
@@ -53,19 +58,23 @@ validate location
         public void Setup()
         {
             _output.WriteLine(Messages.Title);
-            InitialiseMap();
-            InitialiseRover();
+            InitialiseMap(); 

```

Comment:
It's good practice to have a quick glance at the PR before requesting review and/or do a `git diff` before each commit to catch small things like this extra space or missing EOF newlines.